src/core/cluster.py

```diff
@@ -173,10 +175,12 @@ def balancer(self) -> PeerCluster:
             else {}
         )
 
-        if self.runs_broker:  # must be requiring, initialise with necessary broker data
+        if self.runs_broker:  # must be providing, initialise with necessary broker data
```

Comment:
**important thought** what if we use a `cluster_manager` property, which identifies the center of the star (as also discussed in the spec)?

The we can of course have the flag definition with:

```
def cluster_manager(self):
    return self.runs_broker
```

But in this way the code that we write can be easily changed, since probably in Kafka 4, the condition may because based on controller (or we may want to allow the user to define who is the center of the star). So we are already decoupling the specific implementation on 3 and 4. I'm thinking, if we had use the `cluster_manager` role already, ideally this PR would have been just re-writing the `cluster_manager` implementation above with a different condition

Reply:
I'm apprehensive about adding that as part of this ticket, without a strict definition of constraints.
Are we OK with 'anything' being a `cluster_manager`? Does it have to also be a broker?

If 'anything', then this work will need re-doing.
If it's a 'broker', then supporting `cluster_manager` as the center is a 2-line change.

Given that we don't have anything that looks like a `cluster_manager` now, I'd say let's not add code that will need removing later.

Reply:
The initial spec says `cluster_manager` _has_ to be a broker. When we tackle large deployments, then cluster manager can be anything